bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -129,6 +129,12 @@ impl LockTime {
 
     /// Constructs a `LockTime` from `n`, expecting `n` to be a valid block height.
     ///
+    /// ## Note on Locktime and Block Inclusion:
+    ///
+    /// If the current block height is `h` and the locktime is set to `h`,
+    /// the transaction can be included in block `h+1` or later.
+    /// It is safe to broadcast the transaction at block height `h`.
```

Comment:
What does it mean "safe to broadcast"? You can't lose money by broadcasting earlier. The worst thing I can think of is nodes banning you for trying to broadcast invalid tx. Perhaps it should say "possible" instead of "safe".

Reply:
But here we want to document the circumstance when we can broadcast a locktime-constraint tx without breaking any consensus.

so IMO , `safe to broadcast` will make more sense as compared to `possible to broadcast` .
But anyways , I still agree with `possible` .

Reply:
I think `possible to broadcast` is more correct, and both are equally clear when it comes to meaning (in general, Bitcoin has almost no data which is broadcastable but "unsafe" to do so in a security sense). So we should switch to `possible`.